sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -1096,8 +1096,8 @@ def get_container_image_from_options(pipeline_options):
       str: Container image for remote execution.
   """
   worker_options = pipeline_options.view_as(WorkerOptions)
-  if worker_options.worker_harness_container_image:
```

Comment:
"worker_options.worker_harness_container_image" comes from Dataflow v1beta3 API, I don't think you can change that unless you update that API.

Reply:
I think specifically this one is `apache_beam.options.pipeline_options.WorkerOptions` though, not the proto? worker_options is the PipelineOptions object viewed as WorkerOptions.

Reply:
Yeah, you are right.